Makefile

```diff
@@ -79,7 +81,11 @@ lint:
 presto-server-rpm.rpm:
 	wget 'https://repository.sonatype.org/service/local/artifact/maven/content?r=central-proxy&g=com.facebook.presto&a=presto-server-rpm&e=rpm&v=RELEASE' -O $@
 
-smoke: clean-test-all test-images
+smoke: clean-test-all test-images _smoke
+
+smoke-configurable-cluster: clean-test-all _smoke
```

Comment:
I care more that `clean-test` only cleans the things from `make test` than that this doesn't clean more than it needs to (I sometimes run `make test` while in the middle of debugging a product test).   I'm not bothered by adding another clean target, since it matches the extra test targets, but if you want to leave this as is, that's fine too.